src/toolkit.ts

```diff
@@ -40,7 +40,9 @@ export class Toolkit {
   constructor(opts: ToolkitOpts = {}) {
     this.github = new GitHub({token: opts.githubToken});
     this.buildx = new Buildx();
-    this.buildxInstall = new BuildxInstall();
+    this.buildxInstall = new BuildxInstall({
+      gitAuthToken: process.env.GIT_AUTH_TOKEN || opts.githubToken
+    });
```

Comment:
Related to my previous comment we could just have smth like:

```suggestion
    this.buildxInstall = new BuildxInstall({github: this.github});
```